change/@microsoft-fast-foundation-d212c681-4c8d-4516-b25b-be75742f21b1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "fix: adds updates to tabs to support info-pane",
```

Comment:
Infopane is an internal thing for a specific MSFT product - I think the intent here is to support `carousel` like behavior; can we update this to reflect that rather than a specific implementation?